src/plugins/console/public/application/containers/editor/monaco/monaco_editor_actions_provider.ts

```diff
@@ -356,7 +356,7 @@ export class MonacoEditorActionsProvider {
       });
       const lineTokens = getLineTokens(lineContent);
       // if there is 1 or fewer tokens, suggest method
-      if (lineTokens.length <= 1) {
+      if (lineTokens.length <= 1 && !shouldTriggerSuggestions(lineContent)) {
```

Comment:
I think the helper function `shouldTriggerSuggestions` is a bit confusing here, it is used to check the line content if the suggestions should be triggered manually after a specific keybord event, for example backspace, as opposed to the automatic suggestions after "trigger characters". 
The function `getAutocompleteType` here is used when the suggestions are triggered (automatic or manually) to decide for which of 4 item types to look for suggestions. 
For this case I would recommend creating a separate helper or a reg ex, probably one to check that the line is either empty or have a character like `g` or `p` or maybe allow letters/numbers but not punctuation.